packages/manager/modules/telecom-dashboard/README.md

```diff
@@ -0,0 +1,31 @@
+# telecom-dashboard
+
+## Install
+
+```sh
+yarn add @ovh-ux/manager-telecom-dashboard
+```
+
+## Usage
+
+```js
+import 'angular';
+import ovhManagerTelecomDashboard from '@ovh-ux/manager-telecom-dashboard';
+
+angular
+  .module('myApp', [
+    ovhManagerTelecomDashboard,
+  ]);
+```
+
+## Documentation
+
+* Documentation OVH Telecom — [https://docs.ovh.com/fr/telecom/](https://docs.ovh.com/fr/telecom/)
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

Comment:
```suggestion
Always feel free to help out! Whether it's [filling bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

Reply:
This sentence doesn't have any typo:)

Reply:
Apparently both are true and `to file` is more used so good to know